Makefile

```diff
@@ -320,7 +320,7 @@ install-deps-ALPINE-for-pysemgrep:
 # - pkg-config?
 # - coreutils?
 # - gettext?
-BREW_DEPS=pcre gmp pkg-config coreutils gettext
```

Comment:
add a coupling: comment saying to also update setup-m1-buiilder.sh
(actually would be better if setup-m1-builder.sh would use make to install the packages.

Reply:
going top change it to that